### ISSUE TITLE
Theme system

### ISSUE THREAD

**Issue by g0shed** [advanced-data-machines/tailwindcss-vue#17]
Is it possible to get some explanation on the theme system? And how to change it etc?

**Comment by jgraham44** [comment]
Hey, thank you for your interest in the project. Are you able to clone and run the project locally? During the install process you are able to pass in your own theme object that contains the properties needed to style it your own way.

**Comment by g0shed** [comment]
> Hey, thank you for your interest in the project. Are you able to clone and run the project locally? During the install process you are able to pass in your own theme object that contains the properties needed to style it your own way.

yes, I understand, but If I'm using for example just the Table component it starts complaining about not having the theme, i basically have to pass the theme-override prop every time i call a component

**Comment by jgraham44** [comment]
Do you have an example of how you are including it?

**Comment by g0shed** [comment]
in nuxt for example, I pull a component like this:

`import { Button } from '@advanced-data-machines/tailwindcss-vue';`
```
...
components: {
    Button,
},
...
```

**Comment by jgraham44** [comment]
The best way to handle that is just install the specific component and override the [theme](https://github.com/advanced-data-machines/tailwindcss-vue/tree/master/src/themes/default).

```
import { Button } from '@advanced-data-machines/tailwindcss-vue';
const custom = {
    theme: {
        TvButton: {
            ...
        }
    }
};
Vue.use(Button, custom);
```

Can also install just an array of components.

```
import Twv, { Button } from '@advanced-data-machines/tailwindcss-vue';

const components = [Button];
const custom = {
    theme: {
        TvButton: {
            ...
        }
    },
    components
};
Vue.use(Twv, custom);
```
This way you handle the config once.

Let me know if you have other issues or errors.
